### PR TITLE
Added cli code for "snapshot delete"

### PR DESCRIPTION
**Pull request by swemarx**
Needed this for myself, might as well push it upstream.

Was pleasantly surprised that the client already contained DeleteSnapshot(), all I did was implement the cli-stuff.

### CODE REVIEW

File: pkg/cli/snapshot.go [hunk 1]
```diff
@@ -286,3 +304,32 @@ var cmdSnapshotCreate = &cobra.Command{
...
+			fmt.Printf("Could not retrieve required argument: repository. Error: %s\n", err)
+			os.Exit(1)
+		}
+
+		err = v.DeleteSnapshot(repository, snapshotName)
+		if err != nil {
+			fmt.Printf("Error while deleting snapshot. Error: %s\n", err)
+			os.Exit(1)
+		}
+		fmt.Println("Snapshot operation started.")
```
Comment (nickcanz) [inline pkg/cli/snapshot.go:333]: Whoops, could you add a more informative output here? 😄

**Review by nickcanz — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just requested a print statement be updated and then good to go.

File: pkg/cli/snapshot.go (hunk 1 above)
Comment (swemarx) [reply pkg/cli/snapshot.go]: Done 👍🏼

**Review by nickcanz — APPROVED** [review]
Thanks!

**Pull request merged**